docs/source/index.rst

```diff
@@ -40,6 +40,7 @@ The explicit code interfaces and structure are described in the :ref:`api`.
    reference/index
    development/index
    development/doc-overview
+   development/cdf_guide.rst
```

Comment:
This shows up in the header of the page when it is here. I think it might make more sense to put it under the development index.rst file.